kalite/distributed/templates/distributed/base.html

```diff
@@ -302,6 +291,22 @@
                                 </li>
                               </ul>
                             </li>
+                            <!-- Language selection -->
+                            {% if request.session.facility_user.is_teacher %}
+                            <li class="teacher-only">
+                            {% else %}
+                            <li class="not-admin-only">
+                            {% endif %}
```

Comment:
No @MCGallaspy - as discussed with @jamalex and @aronasorman - the language dropdown will not be shown for the admins.